doc/conf.py

```diff
@@ -228,10 +228,10 @@
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'python': ('https://docs.python.org/3', None),
                        'numpy': ('https://numpy.org/doc/stable/', None),
-                       'sklearn': ('https://scikit-learn.org/stable/', None),
+                       'sklearn': ('https://scikit-learn.org/1.5/', None),
```

Comment:
Even though this PR enables support for sklearn 1.6, that doesn't help here because our docs still build on Python 3.8 (because of the tensorflow dependency for Deep IV), where sklearn 1.5 is the latest version.

We need to specifically link to this version of the sklearn docs because there was a change in the structure of their docs between 1.5 and 1.6, so the mismatch between what we were building against vs. what we were linking to was causing broken links.